masonry/src/contexts.rs

```diff
@@ -483,12 +483,36 @@ impl<'w> QueryCtx<'w> {
 // Methods on MutateCtx, EventCtx, and LifeCycleCtx
 impl_context_method!(MutateCtx<'_>, EventCtx<'_>, LifeCycleCtx<'_>, {
     /// Request a [`paint`](crate::Widget::paint) pass.
-    pub fn request_paint(&mut self) {
+    ///
+    /// This doesn't request an [`accessibility`](crate::Widget::accessibility) pass,
+    /// which usually isn't what you want.
```

Comment:
I find this wording confusing with the multiple negatives. Same for the function below.